### DISCUSSION TITLE
New source request

### DISCUSSION THREAD

**Discussion by Warbelian** [dailydotdev/daily#859] [General]
Hi,

Can you evaluate ZD Net and see if it's able to make it into the source list ?

https://www.zdnet.com

A bit about ZD Net:

ZDNet is a business technology news website that covers a range of topics including cybersecurity, cloud computing, mobile devices, software, and internet of things (IoT). It provides in-depth news, analysis, and commentary on the latest developments in the tech industry, as well as reviews of new products and services. With a global network of reporters and editors, ZDNet aims to provide comprehensive coverage of the tech industry from multiple perspectives. The website is aimed at IT professionals, business leaders, and anyone interested in staying up-to-date with the latest technology news and trends.

Many thanks
Carlos

**Comment by idoshamun** [comment]
Hi @Warbelian,

Thanks for the suggestion.
Can you please suggest a source via the app? (https://docs.daily.dev/docs/for-content-creators/suggest-new-source#how-to-submit-a-new-source-request)
This would be the best way to ensure we review it properly and grant you reputation in case we decide to approve it.

**Comment by Warbelian** [comment]
Hi @idoshamun 

Many thanks for the guide, will do it that way instead.
